src/Compilers/CSharp/Test/Symbol/Symbols/SymbolErrorTests.cs

```diff
@@ -880,6 +880,71 @@ interface i1
                 Diagnostic(ErrorCode.ERR_EventPropertyInInterface, "add"));
         }
 
+        [Fact]
+        public void CS0065ERR_EventNeedsBothAccessors_Interface05()
+        {
+            var text = @"
+public interface I2 { }
+
+public interface I1
+{
+    event System.Action I2.P10;
+}
+";
+            CreateCompilation(text).VerifyDiagnostics(
+                // (6,27): error CS0071: An explicit interface implementation of an event must use event accessor syntax
+                //     event System.Action I2.P10;
+                Diagnostic(ErrorCode.ERR_ExplicitEventFieldImpl, ".").WithLocation(6, 27),
+                // (6,31): error CS1514: { expected
+                //     event System.Action I2.P10;
+                Diagnostic(ErrorCode.ERR_LbraceExpected, ";").WithLocation(6, 31),
+                // (6,31): error CS1055: An add or remove accessor expected
+                //     event System.Action I2.P10;
+                Diagnostic(ErrorCode.ERR_AddOrRemoveExpected, ";").WithLocation(6, 31),
+                // (7,2): error CS1513: } expected
+                // }
+                Diagnostic(ErrorCode.ERR_RbraceExpected, "").WithLocation(7, 2),
+                // (6,28): error CS0541: 'I1.P10': explicit interface declaration can only be declared in a class or struct
+                //     event System.Action I2.P10;
+                Diagnostic(ErrorCode.ERR_ExplicitInterfaceImplementationInNonClassOrStruct, "P10").WithArguments("I1.P10").WithLocation(6, 28),
+                // (6,28): error CS0065: 'I1.P10': event property must have both add and remove accessors
+                //     event System.Action I2.P10;
+                Diagnostic(ErrorCode.ERR_EventNeedsBothAccessors, "P10").WithArguments("I1.P10").WithLocation(6, 28));
+        }
+
+        [Fact]
+        public void CS0065ERR_EventNeedsBothAccessors_Interface06()
```

Comment:
>CS0065ERR_EventNeedsBothAccessors_Interface06 [](start = 20, length = 45)

We probably should test more variations with different whitespace between dot and the following identifier and with lack or presence of ```;```